weekly_reports.rb

```diff
@@ -36,6 +36,7 @@ def all_projects(date, slack, text, rerun, verbose, customer_facing)
       error = <<~MSG
       Generation of weekly report for project #{project.name} stopped due to error:
       #{e}
+      #{e.error_messages.join("\n")}
```

Comment:
Sorry, final thing - you're joining them here with an `"\n"` but you already added `"\n"` to the messages themselves, so when printed there are some big gaps.